### ISSUE TITLE
Is this can be use on Ubuntu Linux system？

### ISSUE THREAD

**Issue by zkbqhuang** [Slidingwall/synthv-dictionaries#1]
Is this can be use on Ubuntu system？
if can，how to do it.

**Comment by Slidingwall** [comment]
Yes. The `Dicts` folder is in Synthesizer V's installation directory.
Just put *.json file in the right folder just like Windows.
